File: tcg-java/src/test/java/de/kimminich/kata/tcg/syntactic/CardSugar.java
package de.kimminich.kata.tcg.syntactic;

import de.kimminich.kata.tcg.Card;

import java.util.Optional;

public class CardSugar {

    public static Optional<Card> card(int card) {
        return Optional.of(new Card(card));
    }

    public static Optional<Card> noCard() {
        return Optional.empty();
    }

    public static Card aCardWithValue(int value) {
        return new Card(value);
    }

}

File: tcg-kotlin/src/test/kotlin/de/kimminich/kata/tcg/syntactic/CardSugar.kt
package de.kimminich.kata.tcg.syntactic

import de.kimminich.kata.tcg.Card

object CardSugar {
    fun card(card: Int): Card {
        return Card(card)
    }

    fun aCardWithValue(value: Int): Card {
        return Card(value)
    }
}
